changelog/@unreleased/pr-4968.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Transaction#markTableInvolved.
```

Comment:
Might be good to have a bit more detail here (in particular, don't use this unless you really know what you're doing type of warnings)

Reply:
Updated, but without the warning. I don't think anyone really reads these, and if someone were to use this, I'm sure they'd reach out first, or if not, I doubt a warning would make them do it...